components/o3-button/stories/sb-story-config.ts

```diff
@@ -4,15 +4,12 @@ import type { Meta } from '@storybook/react';
 export const TemplateSBConfig: Meta = {
 	argTypes: {
 		theme: {
-			control: {
-				type: 'radio',
-				labels: {
-					inverse: 'inverse',
-					mono: 'mono',
-					standard: 'standard',
-				},
-			},
-			options: ['inverse', 'mono', 'standard'],
+			options: ['default','inverse', 'mono',],
+			mapping: {
+				default: '',
```

Comment:
To align with the terminology we are using in Figma?
```suggestion
			options: ['standard', 'inverse', 'mono'],
			mapping: {
				standard: '',
```